internal/config/config.go

```diff
@@ -229,7 +243,18 @@ func (o *Override) Parse() error {
 		if strings.HasSuffix(typename, "-go") {
 			typename = typename[:len(typename)-len("-go")]
 		}
-		o.GoPackage = o.GoType[:lastDot]
+		// Upstream package name assignment
+		//o.GoPackage = o.GoType[:lastDot]
```

Comment:
I left the upstream code as commented out while fiddling around. Those lines should be removed upon an accepted PR to avoid polluting the code.